src/store/notificationSlice.js

```diff
@@ -0,0 +1,127 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { getCookie } from '../utils/cookie';
+import { STATUSES } from '../constants/statuses';
+import {
+  fetchStateHelper,
+  rejectStateHelper,
+} from '../utils/reduxStateHelper';
+
+const initialState = {
+  data: [],
+  status: STATUSES.IDLE,
+  errors: [],
+};
+
+export const fetchNotifications = createAsyncThunk(
+  'expense/fetchNotifications',
+  async () => {
+    const token = getCookie('token');
+    const res = await fetch(
+      `${process.env.REACT_APP_BASE_URL}/notifications/my`,
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${token}`,
+        },
+      }
+    );
+    const data = await res.json();
+
+    return data;
```

Comment:
NB: `data` is unused
```suggestion
    return await res.json();
```